servers/physics_server_3d.h

```diff
@@ -150,6 +150,7 @@ class PhysicsDirectSpaceState3D : public Object {
 	struct RayResult {
 		Vector3 position;
 		Vector3 normal;
+		int face_index;
```

Comment:
This may be worth initializing to `-1` to avoid issues with uninitialized data. (Other types like Vector3 in this struct are default-initialized, so they don't need it.)

Reply:
Added, thanks.